packages/components/organisms/f-checkout/src/components/tests/Checkout.test.js

```diff
@@ -32,6 +36,27 @@ describe('Checkout', () => {
         expect(wrapper.exists()).toBe(true);
     });
 
+    it('should show the login link.', () => {
```

Comment:
Hyper mega ultra nit: please remove the `.` at the end (again, to keep it consistent) 😄

Reply:
Not sure, but should we place this test within a `describe` so it's grouped like the rest of the tests?